b2sdk/session.py

```diff
@@ -30,13 +30,31 @@ def __getattr__(self, name):
         @functools.wraps(f)
         def wrapper(*args, **kwargs):
             auth_failure_encountered = False
+            token_type = getattr(f, 'token_type', TokenType.API)
```

Comment:
please add a `*magic*` comment here with explanation what happens. I think this piece of code is brilliant, but all of those are documented specifically in this project.

Reply:
pleeeease add "*magic*" in the comment

Reply:
you still have not added it ;) It should be added because documentation tells the new developer to `git grep '*magic*'` to find all magic places.